src/Microsoft.OData.Core/Metadata/EdmLibraryExtensions.cs

```diff
@@ -601,6 +601,11 @@ internal static bool IsPrimitiveType(Type clrType)
                 return true;
             }
 
+            if (clrType == typeof(System.DateTime))

```

Comment:
Any reason to use a fully qualified name here?

Reply:
@paulodero what is the implication of using a fully qualified name?

Reply:
@ElizabethOkerio Going by [C# naming conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#naming-conventions) you do not have to use `FQN` if the namespace is already imported. In this case `System `is already imported and its redundant to do this. You only use FQN in simple code that do not include `using directives` or when you want to resolve namespace conflict.